### DISCUSSION TITLE
In einem YForm E-Mail Template die Labels eines Choice Feldes nutzen?

### DISCUSSION THREAD

**Discussion by tyrant88** [FriendsOfREDAXO/.github#6] [Q&A] [answered]
Ich möchte, dass in der vom Formular versendeten E-Mail nicht der WERT eines Choice-Feldes, sondern das LABEL steht, also:

`choice|sex|Geschlecht|Weiblich=1,Männlich=2,Divers=3|1|0|`

`REX_YFORM_DATA[field="sex"] ` liefert aber nur "1".

**Comment by tyrant88** [comment]
habe so ein Gefühl, @dtpop kennt die Antwort...

**Comment by dtpop** [comment, marked as answer]
Ich zitiere mal aus der Hilfe in YForm:

Beispiel E-Mail

REX_YFORM_DATA[field="choice"]
REX_YFORM_DATA[field="choice_LABELS"]
REX_YFORM_DATA[field="choice_LIST"]

Tipp: Bei E-Mails werden über das Feld-Suffix _LABELS direkt die Beschriftungen als kommaseparierter Text bzw. _LIST mit Zeilenumbrüchen zurückgegeben.

**Reply by tyrant88** [reply]
Bei mir sieht das in einer E-Mail so aus:

Bei "choice_LABELS":
Büro Frankfurt

Bei "choice_LIST":
[ ] Büro Gießen
[⨉] Büro Frankfurt
[ ] Büro München


Coole Sache.
